GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -910,30 +950,27 @@ void FramebufferManagerCommon::NotifyRenderFramebufferSwitched(VirtualFramebuffe
 	NotifyRenderFramebufferUpdated(vfb);
 }
 
-void FramebufferManagerCommon::NotifyVideoUpload(u32 addr, int size, int width, GEBufferFormat fmt) {
+void FramebufferManagerCommon::NotifyVideoUpload(u32 addr, int size, int stride, GEBufferFormat fmt) {
 	// Note: UpdateFromMemory() is still called later.
 	// This is a special case where we have extra information prior to the invalidation.
 
 	// TODO: Could possibly be an offset...
-	VirtualFramebuffer *vfb = GetVFBAt(addr);
+	// Also, stride needs better handling.
+	VirtualFramebuffer *vfb = ResolveVFB(addr, stride, fmt);
```

Comment:
I think the stride/width thing was to fix #8392.  So I guess that's what might be broken, but maybe it's fine as long as we would create a new framebuf... was some time ago.

-[Unknown]